src/test/store/receive-profile.js

```diff
@@ -0,0 +1,139 @@
+import { assert } from 'chai';
```

Comment:
Thank you, these look great! Did you need me to look at the rest of the files too?